src/_tutorials/server/cmdline.md

```diff
@@ -64,7 +64,7 @@ Let's run a small program.
 
  1. Create a file called `helloworld.dart` that contains this code:
 
-    <?code-excerpt "misc/test/samples_test.dart (hello-world)"?>
```

Comment:
Oh wait a minute... I didn't realize that this code snippet was already tested and in the build.